setup.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
```

Comment:
Ignore this file. Depends on PR:
https://github.com/seintun/proj_pi_server/pull/15

I'm waiting until the other PRs are approved before merging mines, since I'm good at dealing with or avoiding merge conflicts